tests/GeneralStateTestsRunner.js

```diff
@@ -43,116 +43,126 @@ function parseTestCases (forkConfig, testData, data, gasLimit, value) {
   return testCases
 }
 
-function runTestCase (options, testData, t, cb) {
+function runTestCase(options, testData, t, cb) {
   const state = new Trie()
   let block, vm
 
-  async.series([
-    function (done) {
-      var VM
-      if (options.dist) {
-        VM = require('../dist/index.js').default
-      } else {
-        VM = require('../lib/index').default
-      }
-      vm = new VM({
-        state: state,
-        hardfork: options.forkConfig.toLowerCase()
-      })
-      testUtil.setupPreConditions(state, testData, done)
-    },
-    function (done) {
-      var tx = testUtil.makeTx(testData.transaction, options.forkConfig.toLowerCase())
-      block = testUtil.makeBlockFromEnv(testData.env)
-      tx._homestead = true
-      tx.enableHomestead = true
-      block.isHomestead = function () {
-        return true
-      }
-
-      if (!tx.validate()) {
-        return done()
-      }
+  async.series(
+    [
+      function(done) {
+        var VM
+        if (options.src) {
```

Comment:
@evertonfraga Sorry for the formatting I guess the whole file got prettified. Here is where I changed the default to test `dist`, and accepts flag `--src` to test against the source (`lib` folder).